prusti-specs/src/export_spec_rewriter.rs

```diff
@@ -0,0 +1,336 @@
+use proc_macro2::{TokenStream, TokenTree};
+use quote::{quote, quote_spanned, ToTokens};
+use syn::spanned::Spanned;
+
+// generate a hash of path
+fn hash_path(path: syn::Path) -> u64 {
+    use std::collections::hash_map::DefaultHasher;
+    use std::hash::{Hash, Hasher};
+    println!("path to hash: {:?} \n", &path);
```

Comment:
I know this is still WIP but this should eventually become `debug!` (or just removed entirely).